application/server/server_smoke_test.go

```diff
@@ -1023,6 +1023,36 @@ func Test_SmokeSnykCodeDelta_NoNewIssuesFound_JavaGoof(t *testing.T) {
 	assert.Equal(t, 0, len(issueList), "no issues expected, as delta and no new change")
 }
 
+func Test_SmokeScanUnmanaged(t *testing.T) {
+	c := testutil.IntegTest(t)
+	testsupport.NotOnWindows(t, "git clone does not work here. dunno why. ") // FIXME
+	loc, jsonRPCRecorder := setupServer(t, c)
+	c.SetSnykOssEnabled(true)
+	c.SetSnykIacEnabled(false)
+	c.EnableSnykCodeQuality(false)
+	cleanupChannels()
+	di.Init()
+
+	cloneTargetDir, err := storedconfig.SetupCustomTestRepo(t, types.FilePath(t.TempDir()), testsupport.CppGoof, "259ea516a4ec", c.Logger())
+	cloneTargetDirString := string(cloneTargetDir)
+	if err != nil {
+		t.Fatal(err, "Couldn't setup test repo")
+	}
+
+	initParams := prepareInitParams(t, cloneTargetDir, c)
+	folderConfig := c.FolderConfig(cloneTargetDir)
+	folderConfig.AdditionalParameters = []string{"--unmanaged"}
```

Comment:
this may be flaky. the right way would be to add it to the stored config in the target directory, I guess.